src/main/java/io/goobox/sync/storj/App.java

```diff
@@ -265,7 +265,7 @@ public void onError(int code, String message) {
                     }
                 });
             } catch (KeysNotFoundException e) {
-                logger.info("No keys found. Waiting for keys to be imported.");
+                logger.error("keyfile not found at the expected dir " + Utils.getDataDir().toFile());
```

Comment:
Could you not change the message completely, but just add the location?
i.e. `logger.info("No keys found at {}. Waiting for keys to be imported.", location);`